backstop/tests/text-field.html

```diff
@@ -278,9 +298,13 @@ <h2>Without label</h2>
           class="iui-input"
         />
         <div class="iui-message">
-          <!-- Use error.svg from generic-icons package. -->
-          <svg aria-hidden="true">
-            <path d="M8,0a8,8,0,1,0,8,8A8,8,0,0,0,8,0ZM9,12H7V10H9ZM9,9H7V4H9Z" />
+          <!-- Use status-error.svg from @itwin/itwinui-icons package. -->
+          <svg
+            aria-hidden="true"
+            viewBox="0 0 16 16"
+            title="Negative message."
```

Comment:
Can you actually see the title on hover?

Reply:
They are not appearing for me.  I can remove them.

Reply:
Could you remove all titles from svgs as they are not really working??
Leftovers in tile.html

Reply:
Removed title from SVGs & replaced titles on buttons with `aria-label`.